config/locales/en.yml

```diff
@@ -296,6 +296,7 @@ en:
   # /my-account
   my_account:
     title: My account
+    name_information: This is the name that will appear on you end of module certificate. You can use this setting to change how your name appears. Changing your name on this account will not affect your Gov.UK One Login account.
```

Comment:
I believe that the GOV.UK One Login is always capitalised i.e. the GOV part

Reply:
[4a353bc](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/977/commits/4a353bc8a613d1de1fb9e6b9ad26f6be4bf7a54d)